tests/track/test_track.py

```diff
@@ -0,0 +1,221 @@
+import shutil
+from filecmp import cmpfiles
+from pathlib import Path
+
+import pytest
+
+from OTVision.config import CONFIG
+from OTVision.helpers.files import get_files
+from OTVision.track.track import main as track
+
+SIGMA_L = CONFIG["TRACK"]["IOU"]["SIGMA_L"]
+SIGMA_H = CONFIG["TRACK"]["IOU"]["SIGMA_H"]
+SIGMA_IOU = CONFIG["TRACK"]["IOU"]["SIGMA_IOU"]
+T_MIN = CONFIG["TRACK"]["IOU"]["T_MIN"]
+T_MISS_MAX = CONFIG["TRACK"]["IOU"]["T_MISS_MAX"]
+
+
+def define_and_create_folders(
+    test_data_tmp_dir: Path, test_data_dir: Path, dir_name: str
+) -> tuple[Path, Path]:
+    test_track_dir = test_data_dir / "track" / dir_name
+    (test_data_tmp_dir / "track").mkdir(exist_ok=True)
+    test_track_tmp_dir = test_data_tmp_dir / "track" / dir_name
+    test_track_tmp_dir.mkdir(exist_ok=True)
+    return test_track_dir, test_track_tmp_dir
+
+
+def get_reference_data(test_track_dir: Path) -> tuple[list[Path], list[Path]]:
+    ref_detections_files = get_files(
+        paths=[test_track_dir],
+        filetypes=[CONFIG["DEFAULT_FILETYPE"]["DETECT"]],
+    )
+    ref_tracks_files = get_files(
+        paths=[test_track_dir], filetypes=[CONFIG["DEFAULT_FILETYPE"]["TRACK"]]
+    )
+
+    return ref_detections_files, ref_tracks_files
+
+
+def copy_input_data_to_tmp_folder(
```

Comment:
Same as above, turn setup method for each file into a pytest fixture. Note and make use that fixtures can also depend on other fixtures~